cmd/slinky-config/main.go

```diff
@@ -422,13 +424,27 @@ func createMarketMap() error {
 			providers := tickersToProviders[tickerStr].Providers
 			providers = append(providers, config)
 			tickersToProviders[tickerStr] = mmtypes.Providers{Providers: providers}
+
+			if _, ok := tickersToPaths[tickerStr]; !ok {
+				tickersToPaths[tickerStr] = mmtypes.Paths{}
+			}
+			paths := tickersToPaths[tickerStr].Paths
+			paths = append(paths, mmtypes.Path{Operations: []mmtypes.Operation{
```

Comment:
We're medianizing over all paths, no? This would give extra weight to the raw feed if it alr existed?

Reply:
It's just a pure median over all feeds with optional 1 hop inversion.